app/dependency_file_fetchers/node.rb

```diff
@@ -5,7 +5,7 @@ class Node < Base
     def files
       @files ||= [
         fetch_file_from_github("package.json"),
-        fetch_file_from_github("npm-shrinkwrap.json")
+        fetch_file_from_github("yarn.lock")
```

Comment:
Sweet, probably add something to the README if not already there that let's people know that we only support yarn?